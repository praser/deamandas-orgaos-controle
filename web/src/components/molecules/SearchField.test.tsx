import React from 'react';
import { render, screen } from '@testing-library/react';
import SearchField from './SerchField';

describe('SearchField molecule component', () => {
  let searchField: HTMLElement;

  beforeEach(() => {
    render(<SearchField />);
    searchField = screen.getByPlaceholderText('Pesquisar...');
  });

  it('is expected to be in the page', () => {
    expect(searchField).toBeVisible();
  });

  it('is expected to match snapshot', () => {
    expect(searchField).toMatchSnapshot();
  });
});
