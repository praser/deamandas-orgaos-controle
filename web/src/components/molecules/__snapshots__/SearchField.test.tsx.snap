// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchField molecule component is expected to match snapshot 1`] = `
<input
  class="sc-bdvvaa hxuJHo"
  placeholder="Pesquisar..."
/>
`;
